health/notifications/awssns/README.md

```diff
@@ -15,29 +16,30 @@ As part of it's AWS suite, Amazon provides a notification broker service called
+-   An Amazon SNS topic to send notifications to with one or more subscribers. The [Getting
     Started](https://docs.aws.amazon.com/sns/latest/dg/sns-getting-started.html) section of the Amazon SNS documentation
-    covers the basics of how to set this up.  Make note of the Topic ARN when you create the topic.
--   While not mandatory, it is highly recommended to create a dedicated IAM user on your account for Netdata to send notifications.  This user needs to have programmatic access, and should only allow access to SNS.  If you're really paranoid, you can create one for each system or group of systems.
+    covers the basics of how to set this up. Make note of the **Topic ARN** when you create the topic.
+-   While not mandatory, it is highly recommended to create a dedicated IAM user on your account for Netdata to send notifications. This user needs to have programmatic access, and should only allow access to SNS. If you're really paranoid, you can create one for each system or group of systems.
 
-Once you have all the above, run the following command as the user Netdata runs under:
+## Enabling Amazon SNS
 
+To enable SNS:
+1. Run the following command as the user Netdata runs under:
 ```
 aws configure
 ```
+2. Enter the access key and secret key for accessing Amazon SNS. The system also promts you to enter the default region and output format, but you can leave those blank because Netdata doesn't use them.
 
-THis will prompt you for the access key and secret key for accessing Amazon SNS (as well as the default region and output format, but you can leave those blank because we don't use them).
-
-Once that's done, you're ready to go and can specify the desired topic ARN as a recipient.
-
-Notes:
-
--   Netdata's native email notification support is far better in almost all respects than it's support through Amazon SNS.  If you want email notifications, use the native support, not SNS.
-    -   If you need to change the notification format for SNS notifications, you can do so by specifying the format in `AWSSNS_MESSAGE_FORMAT` in the configuration.  This variable supports all the same variables you can use in custom notifications.
-    -   While Amazon SNS supports sending differently formatted messages for different delivery methods, Netdata does not currently support this functionality.
+3. Specify the desired topic ARN as a recipient, see [SNS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/US_SetupSNS.html#set-up-sns-topic-cli).
+4. Optional: You can change the notification format for SNS notifications in the Netdata `???` configuration file. Specify the format: `AWSSNS_MESSAGE_FORMAT="${status} on ${host} at ${date}: ${chart} ${value_string}"`. This variable supports all the same variables you can use in custom notifications.
```

Comment:
I would suggest just keeping the original phrasing here. We don’t need to provide an example format here, though we should possibly provide the default format (which is what is shown here).